packages/server/providers/couchdb.js

```diff
@@ -90,7 +90,15 @@ const couchDb = (app, io, pouchOptions) => {
         if (err.status === 409) {
           return retryUntilWritten(db, doc)
         } else { // new doc
-          return db.put(doc).then(() => res.send({ msg: 'Saved', data: doc }))
```

Comment:
Is this fixing a bug @lilitkarapetyan ?

Reply:
Yes, this part is for updating "initial_wargame" and I added "settings" if "initial_wargame" doesn't exist.